README.md

```diff
@@ -1,2 +1,18 @@
 # xpdSchema
 Schema for XPD metadatastore documents
+
+##Notes
+An important thing to note is the "server/client" key convention.
+Servers contain the data which the key describes. 
+For example data which has the key `detector_calibration_server_uid`
```

Comment:
For example, calibration data is stored in a `server` which has the key `detector_calibration_server_uid` containing a uid value.  This `server` contains the images/other data needed to calibrate the detector.